helm/dagster/charts/dagster-user-deployments/templates/helpers/_helpers.tpl

```diff
@@ -92,6 +92,21 @@ DAGSTER_K8S_PIPELINE_RUN_NAMESPACE: "{{ .Release.Namespace }}"
 DAGSTER_K8S_PIPELINE_RUN_ENV_CONFIGMAP: "{{ template "dagster.fullname" . }}-pipeline-env"
 {{- end -}}
 
+{{/*
+Whether a deployment should include config in its launched runs (defaults to true if includeConfigInLaunchedRuns.enabled
+is not explicitly set to a false-y value)
+*/}}
+{{- define "dagsterUserDeployments.includeConfigInLaunchedRuns" }}
+{{- if .includeConfigInLaunchedRuns -}}
+{{- if or .includeConfigInLaunchedRuns.enabled (not (hasKey .includeConfigInLaunchedRuns "enabled")) -}}
```

Comment:
When would the key be missing? don't we have schema on this?

I feel like if we have schema on this, we wouldn't need this template. There would only be one case to handle: check whether the deployment has `"includeConfigInLaunchedRuns"` as a key, and that `enabled` is not set to false.